tmux/2023-05-BBN-M2-Tourniquet.yml

```diff
@@ -42,33 +44,20 @@ on_project_start: |
 tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
 
 windows:
-  - datahub: ros2 run ros_tcp_endpoint default_server_endpoint --ros-args
-      -r __ns:=${ROS_NAMESPACE}
-      -p ROS_IP:=0.0.0.0
-  - hl2ss_bridge: ros2 run angel_system_nodes hl2ss_ros_bridge --ros-args
+  # Connection with the HL2 app via HL2SS sensor data streaming and the
+  # ROS-TCP-Endpoint
+  - sensor_input: ros2 run angel_system_nodes redis_ros_bridge --ros-args
       -r __ns:=${ROS_NAMESPACE}
-      -p ip_addr:=${HL2_IP}
+      -p url:=${HL2SS_API_URL}
       -p image_topic:=PVFramesBGR
       -p hand_pose_topic:=HandJointPoseData
-      -p audio_topic:=HeadsetAudioData
-      -p sm_topic:=SpatialMapData
-      -p head_pose_topic:=HeadsetPoseData
-      -p pv_width:=1280
-      -p pv_height:=720
-      -p pv_framerate:=30
-      -p sm_freq:=5
-
-  # Visualize Images being output from the headset
-  - rqt_pv_images: rqt -s rqt_image_view/ImageView
-      --args ${ROS_NAMESPACE}/PVFramesBGR
-      --ros-args -p _image_transport:=raw
 
   - object_detector: ros2 run angel_system_nodes berkeley_object_detector --ros-args
       -r __ns:=${ROS_NAMESPACE}
       -p image_topic:=PVFramesBGR
-      -p det_topic:=ObjectDetections2d
+      -p descriptor_topic:=ObjectDetections2d
```

Comment:
Was this renamed to descriptor_topic somewhere?

Reply:
good catch! I'm not sure what happened. Returning it to `det_topic`.